app/controllers/idv/capture_doc_status_controller.rb

```diff
@@ -15,7 +15,12 @@ def document_capture_session_poll_render_result
       session_uuid = flow_session[:document_capture_session_uuid]
       document_capture_session = DocumentCaptureSession.find_by(uuid: session_uuid)
       return { plain: 'Unauthorized', status: :unauthorized } unless document_capture_session
+      return { plain: 'Cancelled', status: :ok } if document_capture_session.cancelled_at
```

Comment:
One thing I notice is that after the user cancels, this response will cause the page's form to submit, leaving them at the `link_sent` step with the error notice. But since we're not considering cancellation on the step itself, we'll continue to poll, causing the page to reload over and over until the user presumably clicks the "< Back" link to get out of the step.

Possible improvements could be:

- Don't poll if they've cancelled
- Send the user somewhere else after they've cancelled

Reply:
I haven't been able to find a good answer on this one.  We can't stop polling because they might reuse the link we texted and if we route them somewhere else and they reuse the link it won't be polling and won't move them to the next step.  We could invalidate the link once they cancel but that seems harsh.

Reply:
> Shall we defer this improvement to a future story?

That seems okay, yeah. It was mostly noticeable in development, since it's a little slower to load pages. Since we already have some `<meta http-equiv="refresh">` in the app already which acts similar in refreshing pages, I think it'd be okay to leave.